atoma-p2p/src/config.rs

```diff
@@ -51,6 +51,10 @@ pub struct AtomaP2pNodeConfig {
     /// The list of bootstrap nodes to dial
     pub bootstrap_node_addrs: Vec<String>,
 
+    /// The list of bootstrap node peer IDs to connect to
+    /// If not provided, default IPFS bootstrap nodes will be used
+    pub bootstrap_node_peer_ids: Option<Vec<String>>,
```

Comment:
I thought this was captured by `bootstrap_node_addrs`.

Reply:
Nope this is the address independent of the  `PeerID` . We want `PeerIDs` to be fixed, but bootstrap node addresses can change, if we add/remove transports, change the IP address of where it is hosted etc. 

So they are configured seperately